lib/api/v1/controllers/result_test.go

```diff
@@ -96,6 +96,131 @@ var _ = Describe("ResultsController", func() {
+				user := FabricateUser(faker.Email(), faker.Password())
+				result := FabricateResult(user)
+				ctx, response := MakeJSONRequest("GET", fmt.Sprintf("/results/%d", result.ID), nil, nil, user)
+				ctx.Params = append(ctx.Params, gin.Param{Key: "id", Value: fmt.Sprint(result.ID)})
+
+				resultsController := controllers.ResultsController{}
+				resultsController.Show(ctx)
+
+				Expect(response.Code).To(Equal(http.StatusOK))
+			})
+
+			It("returns result detail", func() {
+				user := FabricateUser(faker.Email(), faker.Password())
+				result := FabricateResult(user)
+				ctx, response := MakeJSONRequest("GET", fmt.Sprintf("/results/%d", result.ID), nil, nil, user)
+				ctx.Params = append(ctx.Params, gin.Param{Key: "id", Value: fmt.Sprint(result.ID)})
+
+				resultsController := controllers.ResultsController{}
+				resultsController.Show(ctx)
+
+				Expect(response.Code).To(Equal(http.StatusOK))
+
+				jsonResponse := &serializers.ResultDetailJSONResponse{}
+				GetJSONResponseBody(response.Result(), &jsonResponse)
+
+				Expect(jsonResponse.Data.ID).To(Equal(fmt.Sprint(result.ID)))
+				Expect(jsonResponse.Data.Attributes.CreatedAt).To(Equal(result.CreatedAt.Unix()))
+				Expect(jsonResponse.Data.Attributes.UpdatedAt).To(Equal(result.UpdatedAt.Unix()))
+				Expect(jsonResponse.Data.Attributes.Keyword).To(Equal(result.Keyword))
+				Expect(jsonResponse.Data.Attributes.PageCache).To(Equal(result.PageCache))
+				Expect(jsonResponse.Data.Attributes.Status).To(Equal(result.Status))
+				Expect(jsonResponse.Data.Attributes.UserID).To(Equal(result.UserID))
+			})
+
+			It("returns result relations", func() {
+				user := FabricateUser(faker.Email(), faker.Password())
+				result := FabricateResult(user)
+				adLink := FabricateAdLink(result)
+				link := FabricateLink(result)
+				ctx, response := MakeJSONRequest("GET", fmt.Sprintf("/results/%d", result.ID), nil, nil, user)
+				ctx.Params = append(ctx.Params, gin.Param{Key: "id", Value: fmt.Sprint(result.ID)})
+
+				resultsController := controllers.ResultsController{}
+				resultsController.Show(ctx)
+
+				Expect(response.Code).To(Equal(http.StatusOK))
+
+				jsonResponse := &serializers.ResultDetailJSONResponse{}
+				GetJSONResponseBody(response.Result(), &jsonResponse)
+
+				Expect(jsonResponse.Data.ID).To(Equal(fmt.Sprint(result.ID)))
+				Expect(jsonResponse.Data.Relationships.AdLinks.Data).To(HaveLen(1))
+				Expect(jsonResponse.Data.Relationships.AdLinks.Data[0].ID).To(Equal(fmt.Sprint(adLink.ID)))
+				Expect(jsonResponse.Data.Relationships.AdLinks.Data[0].Type).To(Equal("ad_link"))
+				Expect(jsonResponse.Data.Relationships.Links.Data).To(HaveLen(1))
+				Expect(jsonResponse.Data.Relationships.Links.Data[0].ID).To(Equal(fmt.Sprint(link.ID)))
+				Expect(jsonResponse.Data.Relationships.Links.Data[0].Type).To(Equal("link"))
+			})
+		})
+
+		Context("given result ID that does NOT belong to the authenticated user", func() {
+			It("returns not found error", func() {
```

Comment:
Should it be:
```suggestion
			It("returns error not found", func() {
```

to make it consistent with `returns status ok`?